src/components/play-pause/play-pause.tsx

```diff
@@ -58,7 +58,9 @@ class PlayPause extends Component<any, any> {
     const {eventManager, player} = this.props;
     const playerContainer: HTMLDivElement = document.getElementById(player.config.ui.targetId) as HTMLDivElement;
     eventManager.listen(player, player.Event.UI.USER_CLICKED_PLAY, () => {
-      playerContainer.focus();
+      eventManager.listen(player, player.Event.Core.FIRST_PLAY, () => {
```

Comment:
what if play is called via player API ?
also, does it matter if it was initiated via button click or mouse click ?

Reply:
yes, only for first user's click on the large play button.

Reply:
right, changed it to listenOnce